studies/motordemo/vendordeps/Phoenix6.json

```diff
@@ -1,7 +1,7 @@
 {
     "fileName": "Phoenix6.json",
     "name": "CTRE-Phoenix (v6)",
-    "version": "24.0.0-beta-2",
+    "version": "24.0.0-beta-8",
```

Comment:
if you are somehow still on the beta version of something, you should upgrade to 2024.2.1.